apps/vscode/package.json

```diff
@@ -651,6 +656,13 @@
           "group": "1_editor_mode"
         }
       ],
+      "editor/actions/right": [
+        {
+          "command": "quarto.insertCodeCell",
+          "when": "editorLangId == quarto",
```

Comment:
We don't need this in the Visual Editor (it has its own bar that includes inserting a code cell) or markdown files (executable code cells don't do anything in `.md`).